src/backend/src/controllers/users.controllers.ts

```diff
@@ -96,4 +96,18 @@ export default class UsersController {
       next(error);
     }
   }
+
+  static async updateUserFavorites(req: Request, res: Response, next: NextFunction) {
```

Comment:
see my other comment about moving this stuff but also we should call it `favoriteProject` because this implies that we are updating all the favorites